collections/_infographics/emissions/fossil-fuels-emissions.md

```diff
@@ -1,16 +1,60 @@
 ---
 layout:        infographic
-title:         "Global CO2 emissions: fossil fuels & cement production"
+title:         "Global CO₂ emissions: fossil fuels & cement production"
```

Comment:
Title: I would encourage no colon (to match CZ text and also for flow):

Global CO₂ emissions from fossil fuels & cement production

Reply:
I updated the title (removed colon).

Reply:
As for the text inside the actual Infographic. I don't know if I should be editing the fossil-fuels-emissions.pdf directly.

Maybe @mukrop can provide some input if we want/need to change the pdf itself